src/components/widget/RawWidget.js

```diff
@@ -690,9 +697,45 @@ export class RawWidget extends Component {
             viewId={viewId}
             autoFocus={autoFocus}
             validStatus={widgetData[0].validStatus}
+            isMultiselect={isMultiselect}
           />
         );
 
+      case 'MultiListValue':
+        return (
+          <List
```

Comment:
if the only difference is the `isMultiselect` prop, maybe we can merge this into `listWidgetProps` and use for both single and multi select lists ?

Reply:
This I believe is wise to be done after the BE will provide only the widget type 'MultiListValue'. For now let's leave it as it is pls.